src/js/components/Calendar/Calendar.js

```diff
@@ -630,10 +630,23 @@ const Calendar = forwardRef(
           </Box>
           <Box flex={false} direction="row" align="center">
             <Button
-              a11yTitle={previousMonth.toLocaleDateString(locale, {
-                month: 'long',
-                year: 'numeric',
-              })}
+              a11yTitle={
+                messages?.previous
+                  ? format({
+                      id: 'calendar.previous',
+                      messages,
+                      values: {
+                        date: previousMonth.toLocaleDateString(locale, {
```

Comment:
Seeing this block repeated twice, wondering if there is an opportunity to pull this out as a variable and reference it in both places. Soft opinion here, would also be fine just leaving as is.

```
previousMonth.toLocaleDateString(locale, {
    month: 'long',
    year: 'numeric',
})
```

Reply:
If I keep this method it won't update the aria-label with the messages given.
I thought to create this method:
`const getChevronAriaLabel = (type) => {
      let month = nextMonth;
      if (type === 'previous') {
        month = previousMonth;
      }
      if (messages && messages[type]) {
        return format({
          id: `calendar.${type}`,
          messages,
          values: {
            date: month.toLocaleDateString(locale, {
              month: 'long',
              year: 'numeric',
            }),
          },
        });
      }

      return month.toLocaleDateString(locale, {
        month: 'long',
        year: 'numeric',
      });
    };`
    
What do you think? Type should be equals to 'next'or 'previous'